sdk/eventhubs/azure-messaging-eventhubs/README.md

```diff
@@ -180,21 +179,32 @@ Developers can create a producer by calling `buildProducer()` or `buildAsyncProd
 synchronous `EventHubProducerClient` is created. If `buildAsyncProducer` is used, an asynchronous
 `EventHubProducerAsyncClient` is returned.
 
-Specifying `SendOptions.partitionId(String)` will send events to a specific partition, and not, will allow for automatic
+Specifying `batchOptions.partitionId(String)` will send events to a specific partition, and not, will allow for automatic
 routing. In addition, specifying `partitionKey(String)` will tell Event Hubs service to hash the events and send them to
 the same partition.
 
-The snippet below creates a producer and sends events to any partition, allowing Event Hubs service to route the event
+The snippet below creates a synchronous producer and sends events to any partition, allowing Event Hubs service to route the event
 to an available partition.
 
 ```java
-EventHubProducerAsyncClient client = new EventHubClientBuilder()
+EventHubProducerClient producer = new EventHubClientBuilder()
     .connectionString("<< CONNECTION STRING FOR SPECIFIC EVENT HUB INSTANCE >>")
-    .buildAsyncProducer();
+    .buildProducer();
+EventDataBatch eventDataBatch = producer.createBatch();
+for(EventData eventData : eventDataList) {
```

Comment:
nit: space after `for (` and `if (`.
The indentation is usually 4 spaces rather than 2.

Reply:
Fixed.